src/docfx/lib/log/Progress.cs

```diff
@@ -68,10 +68,11 @@ public static string FormatTimeSpan(TimeSpan value)
 
         private class LogScope : IDisposable
```

Comment:
This could be a struct though, thus it can have fields

```suggestion
        private struct LogScope : IDisposable
```